vpr/src/pack/cluster.cpp

```diff
@@ -671,7 +717,8 @@ static void print_pack_status(int num_clb,
                               int num_molecules_processed,
                               int& mols_since_last_print,
                               int device_width,
-                              int device_height) {
+                              int device_height,
+                              AttractionInfo& attraction_groups) {
     const float print_frequency = 0.04;
```

Comment:
comment what this print frequency is for - Print out a progress message when this fraction of the molecules have been packed.